app/src/main/java/org/jellyfin/androidtv/util/InfoLayoutHelper.java

```diff
@@ -340,12 +344,13 @@ private static void addRatingAndRes(Context context, BaseItemDto item, int media
         if (videoStream != null && videoStream.getWidth() != null && videoStream.getHeight() != null) {
             int width = videoStream.getWidth();
             int height = videoStream.getHeight();
+            Boolean isInterlaced = videoStream.isInterlaced();
             if (width <= 960 && height <= 576) {
                 addBlockText(context, layout, context.getString(R.string.lbl_sd));
             } else if (width <= 1280 && height <= 962) {
-                addBlockText(context, layout, "720");
+                addBlockText(context, layout, "720" + (isInterlaced == null ? "" : isInterlaced ? "i" : "p"));
             } else if (width <= 1920 && height <= 1440) {
-                addBlockText(context, layout, "1080");
+                addBlockText(context, layout, "1080" + (isInterlaced == null ? "" : isInterlaced ? "i" : "p"));
```

Comment:
When isInterlaced is null we should assume it's false, so always show either an `i` for interlaced or `p`.